addons/html_editor/static/src/main/table/table_ui_plugin.js

```diff
@@ -35,6 +35,20 @@ export class TableUIPlugin extends Plugin {
         /** @type {import("@html_editor/core/overlay_plugin").Overlay} */
         this.picker = this.shared.createOverlay(TablePicker, {
             position: "bottom-start",
+            onPositioned: (picker, position) => {
```

Comment:
i think it would be safer to only change the position? something like
 ```
position: this.config.direction === "rtl" ? "bottom-end" : "bottom-start",
```
and remove your code in onpositioned?

Reply:
The problem is that `usePosition`  will always fix the `left`  style and the expand animation will always go to the right and not to the left as expected

Reply:
ok then :)